src/inject/dynamic-theme/modify-css.ts

```diff
@@ -36,6 +36,17 @@ export function getModifiableCSSDeclaration(property: string, value: string, rul
         property === 'stroke' ||
         property === 'stop-color'
     ) {
+        if (value.startsWith('var(')) {
```

Comment:
Please explain what happens here.

Reply:
So if a property is detected that uses an variable as value. The appropriate variable name is being assigned based on it's property. e.g background-color will become darkreader-bg--some-variable. 

Because we don't know which color it will be *due to selectors* we simply parse it to the correct variable and let the browser assign the correct value to the variable. Dark Reader now plays on this to assign the correct values for different variables based on it's value and their desired selectors.